auth-api/tests/unit/api/test_org.py

```diff
@@ -1489,7 +1490,17 @@ def test_approve_org_with_pending_affidavits(client, jwt, session, keycloak_mock
                                content_type='application/json')
     assert org_response.status_code == http_status.HTTP_201_CREATED
 
-    tasks = TaskService.fetch_tasks(task_status=[TaskStatus.OPEN.value], page=1, limit=10)
+    task_search = TaskSearch(
+        None,
+        None,
+        None,
+        None,
+        [TaskStatus.OPEN.value],
+        1,
+        10
+    )
+
+    tasks = TaskService.fetch_tasks(task_search)
```

Comment:
This is an API test.. shouldn't be testing at the service level here.. 

I don't think this belongs in ORG either?